src/wasm-helper.ts

```diff
@@ -6,7 +6,9 @@
 export const id = "/__vite-plugin-wasm-helper";
 
 /* istanbul ignore next */
-const wasmHelper = async (opts = {}, url: string) => {
+const wasmHelper = async (opts = {}, url: string, root: string) => {
+  var inBrowser = !!(typeof window !== "undefined" && window.document && window.document.createElement);
+
   let result: WebAssembly.WebAssemblyInstantiatedSource;
   if (url.startsWith("data:")) {
```

Comment:
Can we reuse the `url` param and use a `file://` URL for SSR?

We can generate code that pass different URL to `initWasm` according to `import.meta.env.SSR`.